optimum/commands/export/openvino.py

```diff
@@ -119,6 +119,14 @@ def parse_args_openvino(parser: "ArgumentParser"):
             "or ['conceptual_captions','laion/220k-GPT4Vision-captions-from-LIVIS','laion/filtered-wit'] for diffusion models."
         ),
     )
+    optional_group.add_argument(
+        "--all-layers",
+        action="store_true",
+        default=None,
+        help=(
+            "Whether embeddings and last MatMul layers should be compressed to a primary precision (usually, INT4)."
```

Comment:
Is there a non-INT4 usecase? I see below that if it is provided, all_layers is set to `None if is_int8 else self.args.all_layers` so it seems like it's ignored for INT8? If it is only for INT4 it would be good to clarify that in the help message. And possibly make it mutually exclusive with INT8.

Reply:
This is for INT4 only.

Reply:
Thanks for your comment! By default we compress those layers to INT8, even if number of bits is set to 4. This flag allow to compress those layer to INT4 as well. I've updated the description, hopefully it is more clear now (since currently the only supported primary precision is INT4, I've rephrased it a bit).